apps/web/stores/NavigationStore.ts

```diff
@@ -50,18 +64,35 @@ export class NavigationStore {
     return "";
   }
 
+  setFarthestIds = (itemId: string, walkthroughId?: string) => {
```

Comment:
What does this mean?

Reply:
It's just a helper function to update both of the stored values for the farthest question and farthest walkthrough the user has currently been to.